ansible/playbook/roles/halkyon/tasks/patch_master.yml

```diff
@@ -0,0 +1,27 @@
+- name: Check if master config is already patched
+  command: cat {{ openshift_env.config_dir }}/master/master-config.yaml
+  register: master_content
+
+- block:
+
+  - name: Generate patch using template
+    template:
+      src: "mutating_webhook_patch.j2"
+      dest: /tmp/mutating_webhook_patch.json
+
+  - name: Get patch file
+    command: cat /tmp/mutating_webhook_patch.json
+    register: patch
+
+  - name: Patch master-config file to use Mutating Webhook
+    command: "oc ex config patch {{ openshift_env.master_config_file }} --patch '{{ patch.stdout }}'"
+    register: r
+
+  - name: Copy patch to new master config file
+    copy: content="{{ r.stdout }}" dest={{ openshift_env.config_dir }}/master/master-config.yaml
+
+  - name: Restart master
+    include_role:
+      name: restart_master
```

Comment:
I dont think that we can use this role as cluster as been started using `oc cluster up` @geoand

Reply:
Which cluster are we talking about?

Reply:
I pushed  a change that should make it work with `oc cluster up` as well. 
The `openshift_env` role is the one that determines what kind of cluster is being used so theoretically it should work

Reply:
Ok. So this role will then perform a `docker restart origin` - https://github.com/snowdrop/openshift-infra/blob/master/ansible/playbook/roles/restart_master/tasks/main.yml#L14

Reply:
Yeah, that's the idea